doc/nrf/ug_ble_controller.rst

```diff
@@ -60,3 +60,6 @@ An exception is the :ref:`ble_llpm` sample, which requires the SoftDevice Contro
 
 By default, all samples are currently configured to use SoftDevice Controller.
 To use the Zephyr Bluetooth LE Controller instead, set :kconfig:option:`CONFIG_BT_LL_SW_SPLIT` to ``y`` in the :file:`prj.conf` file (see :ref:`configure_application`) and make sure to build from a clean build directory.
+
+.. note::
+   If your Bluetooth application requires the LE Secure Connections pairing and you want to use the Zephyr Bluetooth LE Controller, make sure to enable the :kconfig:option:`CONFIG_BT_TINYCRYPT_ECC` option as the ECDH cryptography is not supported by this Bluetooth LE Controller.
```

Comment:
It is required only for nrf5340 but I am not sure if it should be included here.

Reply:
Well in fact according to my experiments, on nRF52 it's the same: when switching to the Zephyr LL, you need to enable the BT_TINYCRYPT_ECC setting, otherwise the LE Secure Connections pairing won't work, because the controller itself doesn't support the required crypto.